packages/ui/package.json

```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "@emotion/styled": "^11.11.0",
     "@jest/globals": "^29.5.0",
-    "@mui/core": "5.0.0-alpha.54",
+    "@mui/base": "5.0.0-beta.11",
```

Comment:
you replaced `core` with `base`, something fishy. it smell like I farted. something smells good or bad I dont kknow

Reply:
sir core package is deprecated and suggests using base